Bottom: c33afdcca73949284834ef4c66bbed62911b9218
Top:    74780731b8bd3e375e587f810cd406ecb64248cd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-07 02:00:46 +0000

Refresh of fix-some-strdup-calls-which-mi

---

diff --git a/src/create.c b/src/create.c
index 4eef584..76a5478 100644
--- a/src/create.c
+++ b/src/create.c
@@ -587,7 +587,7 @@ CreateNewText (LayerTypePtr Layer, FontTypePtr PCBFont,
   text->Direction = Direction;
   text->Flags = Flags;
   text->Scale = Scale;
-  Text->TextString = (TextString && *TextString) ? strdup (TextString) : NULL;
+  text->TextString = (TextString && *TextString) ? strdup (TextString) : NULL;
 
   /* calculate size of the bounding box */
   SetTextBoundingBox (PCBFont, text);
